doc/compiling/windows.md

```diff
@@ -1,68 +1,88 @@
+-   `curl` is highly recommended, as it's required to read the serverlist; `curl[winssl]` is required to use the content store.
+-   `openal-soft`, `libvorbis` and `libogg` are optional, but required to use sound.
+-   `luajit` is optional, it replaces the integrated Lua interpreter with a faster just-in-time interpreter.
+-   `gmp` and `jsoncpp` are optional, otherwise the bundled versions will be compiled
+-   `gettext` is optional, but required to use translations.
 
-Use `--triplet` to specify the target triplet, e.g. `x64-windows` or `x86-windows`.
+### Install dependencies
 
+1. Start up the CMake GUI (Win > search "cmake-gui" > open)
+1. Select **Browse Source...** and select `path/to/minetest` (where you've cloned the repo)
+1. Select **Browse Build...** and select `path/to/minetest/build` (a new folder that CMake will prompt to create)
+1. Select **Configure**
+1. Choose the right Visual Studio version and target platform. Currently, Luanti uses Visual Studio 16 2019, but newer VS versions should work as well. The VS version has to match the version of the installed dependencies.
+1. Choose **Specify toolchain file for cross-compiling**
+1. Click **Next**
+1. Select the vcpkg toolchain file, e.g. `C:/Program Files/Microsoft Visual Studio/2022/Community/VC/vcpkg/scripts/buildsystems/vcpkg.cmake`.
+1. Click Finish
+1. Wait until CMake generates the cache file (this may take about 10-30 minutes, depending on your device)
+1. If there are any errors, solve them and hit **Configure**
+1. Click **Generate**
+1. Click **Open Project**
 
 ## Compile Luanti
 
-### a) Using the vcpkg toolchain and CMake GUI
-
-1. Start up the CMake GUI
-2. Select **Browse Source...** and select DIR/minetest
-3. Select **Browse Build...** and select DIR/minetest-build
-4. Select **Configure**
-5. Choose the right visual Studio version and target platform. It has to match the version of the installed dependencies
-6. Choose **Specify toolchain file for cross-compiling**
-7. Click **Next**
-8. Select the vcpkg toolchain file e.g. `D:/vcpkg/scripts/buildsystems/vcpkg.cmake`
-9. Click Finish
-10. Wait until cmake have generated the cash file
-11. If there are any errors, solve them and hit **Configure**
-12. Click **Generate**
-13. Click **Open Project**
-14. Compile Luanti inside Visual studio.
-
-### b) Using the vcpkg toolchain and the commandline
-
-Run the following script in PowerShell:
-
-```powershell
-cmake . -G"Visual Studio 16 2019" -DCMAKE_TOOLCHAIN_FILE=D:/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release -DENABLE_CURSES=OFF
-cmake --build . --config Release
+There are two ways to compile Luanti: via Visual Studio or via CLI.
+
+### Compile in Visual Studio
+
+14. Compile Luanti inside Visual Studio.
+    -   If you get "Unable to start program '...\x64\Debug\ALL_BUILD'. Access is denied", try compiling via the CLI instead.
+
+### Compile via CLI
+
+While in the `path/to/minetest` folder, run the following command:
```

Comment:
Maybe these sections should also mention that the Install Dependencies step must be completed before these will work.

Reply:
Hmm, I figured folks would go top-down, would numbering the headings help?

Reply:
Maybe it's clear enough. I'm just bringing up the discussion point for consideration, since I can imagine a scenario where someone skips to the part that says "Compile".

Reply:
Numbered the headings and added a lead to each one clarifying that the previous step is required